src/main/resources/deck-configs/dredge/hands.json

```diff
@@ -0,0 +1,146 @@
+[
+  {
+    "containsCards": [
+      "Faithless Looting"
+    ],
+    "excludesCards": [],
+    "containsTags": [
+      "makesGreen",
+      "land"
+    ],
+    "excludesTags": [],
+    "maxKeepSize": 7
```

Comment:
Only keepable because of the Dredge postprocess ye?

Reply:
Actually, postprocess moves us from "keepable hands" to "hands that are likely to work out". Maybe we need to be more selective here.